hack/ipa_debug_tools.sh

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env bash
+
```

Comment:
We should be running with `set -eu` by default.

Reply:
done